src/server/pkg/deploy/assets/assets.go

```diff
@@ -411,7 +427,8 @@ func RethinkRc(backend backend, volume string, hostPath string) *api.Replication
 		spec.Spec.Template.Spec.Volumes[0].PersistentVolumeClaim = &api.PersistentVolumeClaimVolumeSource{
 			ClaimName: rethinkVolumeClaimName,
 		}
-	} else if backend == localBackend {
+	} else if backend == localBackend || backend == microsoftBackend {
+		// ToDo: workaround until https://github.com/pachyderm/pachyderm/issues/960
```

Comment:
Yeah, that's a totally reasonable thing to do for the time being to get things testable.